cli_reference/admin_cli_operations.adoc

```diff
@@ -0,0 +1,171 @@
+= Administrator CLI Operations
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+This topic provides information on the administrator CLI operations and their syntax. You must link:get_started_cli.html[setup and login] with the CLI before you can perform these operations.
+
+[[common-operations]]
+
+== Common Operations
+The CLI allows interaction with the various objects that are managed by
```

Comment:
I know this was just carried over from the basic_cli topic, but maybe it should be more specific here:

s/CLI/administrator CLI/